aws/src/main/java/org/apache/iceberg/aws/glue/GlueTableOperations.java

```diff
@@ -70,6 +73,12 @@
       .orNoop()
       .build();
 
+  // Attempt to set additionalLocations if available on the given AWS SDK version
+  private static final DynMethods.UnboundMethod ADDITIONAL_LOCATIONS = DynMethods.builder("additionalLocations")
+      .hiddenImpl("software.amazon.awssdk.services.glue.model.UpdateTableRequest$Builder", List.class)
```

Comment:
`additionalLocations` should be in `software.amazon.awssdk.services.glue.model.StorageDescriptor`.

Reply:
In that case, we set the storage description only in `IcebergToGlueConverter` class.

Reply:
In that case, we set the `StorageDescriptor` only in `IcebergToGlueConverter` class.